frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts

```diff
@@ -1,6 +1,6 @@
 import { browser, $, ExpectedConditions as until, by, element } from 'protractor';
 
-export const operatorModal = $('.pf-c-modal-box');
+export const operatorModal = $('.co-catalog-page__overlay');
```

Comment:
Is this meant to be part of this PR?

Reply:
Yes when I was running tests (regarding installing pipeline operator) it was failing so I pushed this change along with this pr

Reply:
We should consider a `data-test-id` here, which would be less fragile.

Reply:
I agree with your point, but for this, I want help from someone in openshift dev team on this.
About your first point regarding `Is this meant to be part of this pr` I've edited the pr remove pipeline test from e2e suite for now and included the above `operatorModal` change in my pipeline specific test pr #4147